### PR TITLE
Remove Codecov

### PR DESCRIPTION
**Pull request by dee2716**
This PR is to remove CodeCov due to security breach.
Ref: https://about.codecov.io/security-update/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 7.1
+  - 7.2
```
Comment (hearst-feng) [inline .travis.yml:5]: I don't think we support 7.2...

Comment (dee2716) [reply .travis.yml:5]: Only fre don't support 7.2 tho, right? This package can still support 7.2 I think

Comment (hearst-feng) [reply .travis.yml:5]: would it be pulled into fre?

Comment (dee2716) [reply .travis.yml:5]: This is the travis file. I don't think it will be pulled and used by fre

File: .travis.yml [hunk 2]
```diff
@@ -2,12 +2,16 @@ language: php
...
 
 cache:
   directories:
     - $HOME/.composer/cache/files
     - $HOME/.composer/cache/vcs
 
+before_install:
+  - composer self-update 1.10.21
```
Comment (dee2716) [inline .travis.yml:13]: Added this in before_install stage. Please let me know if I need to address it in a better way. @hearst-feng This is ready for another look

File: .travis.yml (hunk 1 above)
Comment (hearst-feng) [reply .travis.yml:5]: looks like it makes travis build twice?

Comment (dee2716) [reply .travis.yml:5]: It built on 7.1 and 7.2 by adding this. I want to test it's support on 7.2. I can remove it. But I think it only impact the package build itself

**Review by hearst-feng — APPROVED** [review]

**Pull request merged**